components/dashboard/src/data/prebuilds/latest-project-prebuild-query.ts

```diff
@@ -8,19 +8,27 @@ import { useQuery } from "@tanstack/react-query";
 import { prebuildClient } from "../../service/public-api";
 import { Prebuild } from "@gitpod/public-api/lib/gitpod/v1/prebuild_pb";
 import { ApplicationError, ErrorCodes } from "@gitpod/gitpod-protocol/lib/messaging/error";
+import { useCurrentOrg } from "../organizations/orgs-query";
 
 type Args = {
     projectId: string;
 };
 export const useLatestProjectPrebuildQuery = ({ projectId }: Args) => {
```

Comment:
Share comment here because direct comment can't be resolved ()
https://github.com/gitpod-io/gitpod/pull/19326/#pullrequestreview-1837977751